.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma":"none",
+  "tabWidth": 4,
+  "semi": true,
+  "singleQuote": false
```

Comment:
Can we come to a conclusion for usage of single or double quotes? As per standard conventions, using single quote is recommended.
@sshveta need your thoughts as well for finalizing this.

Reply:
@nitishSr Yeah. We need to discuss it.
- You are right - https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#quotes . But I can see the reason is not too strong. 
- But if you read benefit of `""` then I would like to go with it.

Reply:
@sshveta @nitishSr 
- JSON standard does not support a single quote(`' '`).
- Also if we go with the single quote then there will be a huge change in files such as package-lock or other JSON files.
- Hence to make all the files consistent with double-quotes. I am keeping `  "singleQuote": false` for a prettier format.